Add disclaimer for F&F monitors in the monitor template variables section. (#13461)
DataDog/documentation: content/en/monitors/notify/variables.md

_Available for [Log monitors][2], [Trace Analytics monitors][3] (APM), [RUM monitors][4] and [CI Pipeline monitors][5]_

> <div class="alert alert-info"><strong>Note</strong>: This feature is only available for monitors that do not use Formulas & Functions in its query for the moment. It will be available for those monitors in the near future.</div>

Reviewer: ```suggestion
<div class="alert alert-info"><strong>Note</strong>: This feature is only available for monitors that do not use Formulas & Functions in their queries.</div>
```

we don't make any predictions about the future in the docs
